tooling/noirc_artifacts/src/debug.rs

```diff
@@ -38,9 +42,14 @@ impl DebugArtifact {
             .flat_map(|function_symbols| {
                 let brillig_location_maps =
                     function_symbols.brillig_locations.values().flat_map(|brillig_location_map| {
-                        brillig_location_map
-                            .values()
-                            .flat_map(|call_stack| call_stack.iter().map(|location| location.file))
+                        brillig_location_map.values().flat_map(|call_stack_id| {
+                            function_symbols
+                                .location_tree
+                                .get_call_stack(*call_stack_id)
+                                .iter()
+                                .map(|location| location.file)
+                                .collect::<Vec<_>>()
```

Comment:
```suggestion
                                .into_iter()
                                .map(|location| location.file)
```
Same thing from earlier

Reply:
Done